concepts.md

```diff
@@ -0,0 +1,35 @@
+# Concepts
+
+## User
+
+Person identified with [WebID](https://www.w3.org/2005/Incubator/webid/spec/identity/).
```

Comment:
Maybe "Agent" instead of "Person"? Since there can be autonomous agents with WebIDs that are not tied to a particular person.

Reply:
Good point, I think we still may need to distinguish Bots from People, we haven't touched delegation yet but in that approach some autonomous agent (aka Bot) would act on behalf of the person, so Bot would not stay listed in ACL but the person would delegate to it from one's own WebID Profile. I see that quite similar to granting permissions to applications, with possible difference that we may have application 'attended by the user and bot 'unattended'. Actually in #30 @jaxoncreed suggests requiring WebID for applications which might align things even further with delegated approach. Also in our [Use Cases](https://github.com/solid/app-authorization-panel/blob/master/UseCases.md) I included diagram and use case where application has local component and remote component, which we could consider a Bot.
Maybe we will leave *User* to mean person as it usually does and just add Agent, Application and Bot as additional concepts?